templates/template.forTasks.rst

```diff
@@ -5,96 +5,111 @@ Based on: https://dmtn-030.lsst.io/v/DM-7096/index.html#task-topic-type
 Template for Tasks
 ##################
 
-- Summary/context (1 sentence).
+- Summary/context (1-2 sentences).
 
-- Summary of logic/algorithm in a paragaph and/or bullet list.  Be concise and link to other tasks wherever needed.
+- Concise summary of logic/algorithm in a paragaph and/or bullet list.  Be concise and link to other tasks wherever needed.
 
 - Include a sentence about each step, which can be either a:
   
-  - a) retargetable sub-task
+  a) retargetable sub-task
 
-  - b) method within a task.
+  b) method within a task.
 
-- Module membership
 
-Here we state what module implemented the task.
+     
+- Module membership:
+
+  Here we simply state in which module the task is implemented.
 
 See also
 =========
```

Comment:
You can leave out the header "See also" or else devs might get confused about whether they should have a header here in addition to the `.. seealso::` directive.